packages/host/app/components/operator-mode/profile-info-popover.gts

```diff
@@ -137,127 +136,67 @@ export default class ProfileInfoPopover extends Component<ProfileInfoPopoverSign
       </header>
 
       <ProfileInfo />
-      {{! Show credit info if the user has an active plan }}
-      {{#if this.plan}}
-        <div class='credit-info' data-test-credit-info>
-          <div class='info-group'>
-            <span class='label'>Membership Tier</span>
-            <span class='value' data-test-membership-tier>
-              {{#if this.isLoading}}
-                <LoadingIndicator />
-              {{else}}
-                {{this.plan}}
-              {{/if}}
-            </span>
-          </div>
-          <BoxelButton
-            @kind='secondary-light'
-            @size='small'
-            @disabled={{this.isLoading}}
-            data-test-upgrade-plan-button
-            {{on 'click' @toggleProfileSettings}}
-          >Upgrade Plan</BoxelButton>
-          <div class='info-group'>
-            <span class='label'>Monthly Credit</span>
-            <span
-              class={{cn 'value' out-of-credit=this.isOutOfPlanCreditAllowance}}
-              data-test-monthly-credit
-            >
-              {{#if this.isLoading}}
-                <LoadingIndicator />
-              {{else}}
-                <IconHexagon width='16px' height='16px' />
-                {{this.monthlyCreditText}}
-              {{/if}}
-            </span>
-          </div>
-          <div class='info-group additional-credit'>
-            <span class='label'>Additional Credit</span>
-            <span
-              class={{cn 'value' out-of-credit=this.isOutOfCredit}}
-              data-test-additional-credit
-            >{{#if this.isLoading}}
-                <LoadingIndicator />
-              {{else}}
-                <IconHexagon width='16px' height='16px' />
-                {{this.extraCreditsAvailableInBalance}}
-              {{/if}}</span>
-          </div>
-          <div
-            class={{cn 'buy-more-credits' out-of-credit=this.isOutOfCredit}}
-            data-test-buy-more-credits
-          >
+      <WithSubscriptionData as |subscriptionData|>
+        {{! Show credit info if the user has an active plan }}
+        {{#if subscriptionData.hasActiveSubscription}}
+          <div class='credit-info' data-test-credit-info>
+            <div class='info-group'>
+              <span class='label'>Membership Tier</span>
+              {{subscriptionData.plan}}
+            </div>
             <BoxelButton
-              @kind={{if this.isOutOfCredit 'primary' 'secondary-light'}}
-              @size={{if this.isOutOfCredit 'base' 'small'}}
-              @disabled={{this.isLoading}}
-              {{on 'click' @toggleProfileSettings}}
-            >Buy more credits</BoxelButton>
+              @kind='secondary-light'
+              @size='small'
+              @disabled={{or
+                subscriptionData.isLoading
+                this.managePlan.isRunning
+              }}
+              data-test-upgrade-plan-button
+              {{on 'click' (perform this.managePlan)}}
```

Comment:
Why does this have to be a task? Why not simply `href={{this.stripeCustomerPortalLink}}`?

Reply:
Why do you need to test how many times the window open is called?

Reply:
I'm trying to understand, is this because you wanted to test that some of the links will open in a new window? If so you can just assert `target` attribute and trust the browser to do its job.

Reply:
Updated it here [6292d2a](https://github.com/cardstack/boxel/pull/1838/commits/6292d2a6f458c774e5b14abf1089c3a18c85fad0)